app/helpers/template_config/__init__.py

```diff
@@ -0,0 +1,17 @@
+from .business_config import BusinessSurveyConfig
+from .census_config import (
+    CensusNISRASurveyConfig,
+    CensusSurveyConfig,
+    WelshCensusSurveyConfig,
+)
+from .link import Link
+from .survey_config import SurveyConfig
+
+__all__ = [
+    "SurveyConfig",
+    "CensusSurveyConfig",
+    "CensusNISRASurveyConfig",
+    "WelshCensusSurveyConfig",
+    "BusinessSurveyConfig",
+    "Link",
+]
```

Comment:
I think as the folder contains survey configs, it should be named `survey_configs`. We should also consider moving the folder to the top level as it's not a helper.